src/plugins/intel_gpu/src/plugin/ops/result.cpp

```diff
@@ -20,12 +20,9 @@ static void CreateResultOp(ProgramBuilder& p, const std::shared_ptr<ov::op::v0::
     validate_inputs_count(op, {1});
 
     auto prev = op->get_input_node_shared_ptr(0);
-    auto input_id = ov::op::util::get_ie_output_name(op->get_input_source_output(0));
-    if (input_id.empty()) {
-        input_id = prev->get_friendly_name();
-        if (prev->get_output_size() > 1) {
-            input_id += "." + std::to_string(op->get_input_source_output(0).get_index());
-        }
+    auto input_id = prev->get_friendly_name();
+    if (prev->get_output_size() > 1) {
+        input_id += "." + std::to_string(op->get_input_source_output(0).get_index());
```

Comment:
why do we need this legacy logic? Result has its own name, which can be used here (for some reason)